src/content/interop/c-interop.md

```diff
@@ -79,269 +86,296 @@ Hello World
+| [IntPtr][]               | Represents the `intptr_t` type in C.               |
+| [Long][]                 | Represents the `long int` (`long`) type in C.      |
+| [LongLong][]             | Represents the `long long` type in C.              |
+| [Short][]                | Represents the `short` type in C.                  |
+| [SignedChar][]           | Represents the `signed char` type in C.            |
+| [Size][]                 | Represents the `size_t` type in C.                 |
+| [UintPtr][]              | Represents the `uintptr_t` type in C.              |
+| [UnsignedChar][]         | Represents the `unsigned char` type in C.          |
+| [UnsignedInt][]          | Represents the `unsigned int` type in C.           |
+| [UnsignedLong][]         | Represents the `unsigned long int` type in C.      |
+| [UnsignedLongLong][]     | Represents the `unsigned long long` type in C.     |
+| [UnsignedShort][]        | Represents the `unsigned short` type in C.         |
+| [WChar][]                | Represents the `wchar_t` type in C.                |
 
 {:.table .table-striped }
 
+## Generate FFI bindings with `package:ffigen`
 
-## Generating FFI bindings with `package:ffigen`
-
-For large API surfaces it can be time-consuming
+For large API surfaces, it can be time-consuming
 to write the Dart bindings that integrate with the C code.
-To reduce this burden,
-you can use the [`package:ffigen`][ffigen] binding generator
-to automatically create FFI wrappers from C header files.
+To have Dart create FFI wrappers from C header files,
+use the [`package:ffigen`][ffigen] binding generator.
 
-<a id="native-assets"></a>
-## Building and bundling native assets
+## Build and bundle native assets {:#native-assets}
 
 :::note
-The native assets feature is **experimental**,
+The native assets are **experimental**,
 and [in active development]({{site.repo.dart.sdk}}/issues/50565).
 :::
 
-The Native Assets feature aims to resolve a number of issues associated with
+The _Native Assets_ feature should resolve a number of issues associated with
 the distribution of Dart packages that depend on native code.
 It does so by providing uniform hooks for integrating with various
 build systems involved in building Flutter and standalone Dart applications.
 
-The Native Assets feature aims to make it seamless for
-Dart packages to depend on and use native code:
+This feature should simplify how Dart packages depend on and use native code.
+Native Assets should provide the following benefits:
 
-* It builds (if needed) the native code or
-  obtains the binaries using a package's `build.dart` script.
-* It bundles the native [`Asset`][] reported by the `build.dart` script.
-* It makes the native assets available at runtime through
+* Build the native code or obtains the binaries
+  using a package's `build.dart` script.
+* Bundle the native [`Asset`][] that the `build.dart` script reports.
```

Comment:
The new terminology is build hook, `build.dart` hook, or `hook/build.dart`. The hook was moved inside the `hook/` directory, and we don't use "script" anymore.